examples/capture_stream.py

```diff
@@ -18,12 +18,12 @@
     sock.listen()
 
     picam2.encoder = encoder
-    picam2.start_encoder()
-    picam2.start()
 
     conn, addr = sock.accept()
     stream = conn.makefile("wb")
-    picam2.encoder.output = stream
+    picam2.encoder.output = FileOutput(stream)
+    picam2.start_encoder()
+    picam2.start()
     time.sleep(20)
```

Comment:
Thanks for updating these examples. Actually I think the mjpeg_server.py example might need updating too - unfortunately it isn't part of the automated tests (because it runs forever), so you probably wouldn't have realised!

Reply:
Yup, missed that one, tested that after a small change to it